index.js

```diff
@@ -553,14 +554,31 @@ Sharp.prototype.threshold = function(threshold, options) {
   } else {
     throw new Error('Invalid threshold (0 to 255) ' + threshold);
   }
-  
+
   if(typeof options === 'undefined' ||
      options.greyscale === true || options.grayscale === true) {
     this.options.thresholdGrayscale = true;
   } else {
     this.options.thresholdGrayscale = false;
   }
-    
+
+  return this;
+};
+
+/*
+  Automatically remove "boring" image edges.
+    tolerance - if present, is a percentaged tolerance level between 0 and 100 to trim away similar color values
+      Defaulting to 10 when no tolerance is given.
+ */
+Sharp.prototype.trim = function(tolerance) {
+  if (typeof tolerance === 'undefined') {
+    this.options.trimTolerance = 10;
+  } else if (isInteger(tolerance) && inRange(tolerance, 0, 100)) {
```

Comment:
Yes please to a 99% maximum. 100% would suggest the whole image should be trimmed, probably not what most people want :)